syft/__init__.py

```diff
@@ -1 +1,2 @@
 from .tensor import TensorBase
+from .tensor import *
```

Comment:
Also, I think the standalone functions like `syft.matmul`, etc. should go in a separate file.  Like this PyTorch example: https://github.com/pytorch/pytorch/blob/master/torch/functional.py